ansible/roles_ocp_workloads/ocp4_workload_ansible_automation_platform/tasks/main.yml

```diff
@@ -0,0 +1,38 @@
+---
+# Do not modify this file
+
+- name: Running Pre Workload Tasks
+  when: >-
+    ACTION == "create"
+      or ACTION == "provision"
+  ansible.builtin.include_tasks:
+    file: ./pre_workload.yml
+    apply:
+      become: "{{ become_override | bool }}"
+
+- name: Running Workload Tasks
+  when: >-
+    ACTION == "create"
+      or ACTION == "provision"
```

Comment:
Maybe `ACTION in ('create', 'provision')` would be cleaner?

Reply:
Donr